doc/digital-receipt/implementation/digital-receipt-implementation.md

```diff
@@ -0,0 +1,667 @@
+---
+slug: /poscreators/middleware-doc/digital-receipt/implementation/digital-receipt-implementation
+title: 'Digital receipt implementation'
+---
+
+# Digital receipt transmission
+
+>[!IMPORTANT]
+>Before start implementing, please read the getting started section first to make sure you are choosing the best suiting transmitting method for your Point of Sales software. 
+
+fiskaltrust provides two distinct approaches for transmitting receipt requests to the fiskaltrust.Middleware. The first approach is the POS API Helper, which is primarily recommended for testing/sandbox environments and the InStore App. Configuring the POS API Helper within the fiskaltrust.Portal requires minimal implementation effort in your Point of Sale software.
```

Comment:
"Transmitting digital receipts to fiskaltrust", i guess? The POS system is already transmitting them to the Middleware, hopefully :D